common/src/main/java/org/openmrs/analytics/OpenmrsUtil.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.client.api.IClientInterceptor;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.rest.client.interceptor.BasicAuthInterceptor;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Resource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OpenmrsUtil {
+	
+	protected static final Logger log = LoggerFactory.getLogger(OpenmrsUtil.class);
```

Comment:
nit: `private`

Reply:
@bashir2 I'm using the HAPI client to push to the HAPI JPA server in the iSantePlus work, but I think the client you use would depend on the target platform - if there's a platform-specific client, we maybe could use that, and otherwise fall back to the HAPI client for a generic FHIR server?